packages/core-ethereum/crates/core-ethereum-db/src/traits.rs

```diff
@@ -170,6 +170,9 @@ pub trait HoprCoreEthereumDbActions {
     /// Get all channels from a given address.
     async fn get_channels_from(&self, address: &Address) -> Result<Vec<ChannelEntry>>;
 
+    /// Get all the outgoing channels from current node.
+    async fn get_outgoing_channels(&self) -> Result<Vec<ChannelEntry>>;
```

Comment:
We could add also `get_incoming_channels`

Reply:
Will do